### PR TITLE
docs: Fixed packaging issues

### PR DESCRIPTION
**Pull request by david-caro**
Signed-off-by: David Caro <david@dcaro.es>

### CODE REVIEW

File: requirements-docs.txt [hunk 1]
```diff
@@ -1 +1,3 @@
 Sphinx>=1.4.2
+-r requirements-build.txt
+-r requirements.txt
```
Comment (kaplun) [inline requirements-docs.txt]: I think this lines needs to have more spacing.

Comment (david-caro) [inline requirements-docs.txt]: What do you mean?

Comment (kaplun) [inline requirements-docs.txt]: hehe just kidding :+1:

Comment (david-caro) [inline requirements-docs.txt]: Got me :dancers:

**Pull request merged**